python/ccf/clients.py

```diff
@@ -91,8 +90,14 @@ class Identity:
     description: str
 
 
-def int_or_none(v):
-    return int(v) if v is not None else None
+def parse_tx_id(s: Optional[str]):
+    try:
+        if s is not None:
```

Comment:
Since AttributeError is caught, this check isn't necessary.
```suggestion
```

Reply:
That's what I thought, but `mypy` can't tell that:

```
python/ccf/clients.py:96: error: Item "None" of "Optional[str]" has no attribute "split"
Found 1 error in 1 file (checked 1 source file)
```

Reply:
Type checking, type schmecking...